test/skimNtuple2017_HHbtag.cpp

```diff
@@ -3048,11 +3056,11 @@ int main (int argc, char** argv)
 
         if (ele1eta < 2.4)
         {
-          idAndIsoSF_leg1 = myIDandISOScaleFactor[1]->get_ScaleFactor(ele1pt, ele1eta);
+          idAndIsoSF_leg1 = myIDandISOScaleFactor[1]->get_direct_ScaleFactor(ele1pt, ele1eta);
```

Comment:
I have a comment concerning these lines in the three skimmers. in 2018 there is a product of two SF which I assume to be ID and iso. Why this is not the case for 2016 and 2017?

Reply:
I think you did not comment on the right line (ele SF instead of muon), but that's true anyway, I included both id and iso SFs, but did apparently forgot to update their application. I'll correct that in a moment.

Reply:
Fixed in commit cf6d1cd